client/src/components/Button/tests/Button.test.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { shallow } from "enzyme";
+import Button from "../Button";
+
+describe("<Button />", () => {
+  it("renders 1 sub-component", () => {
+    const btn = shallow(<Button />);
+    expect(btn.children.length).toBe(1);
+  });
+
+  it("should render anchor tag button when tag prop is passed", () => {
+    const btn = shallow(<Button tag="a" />);
+    expect(btn.type()).toBe('a');
+  })
+
+  it("should not render anchor tag button when tag prop is missing", () => {
+    const btn = shallow(<Button />);
+    expect(btn.type()).toBe('a');
```

Comment:
`not.toBe` or `toBe`

Reply:
Should have been checking button tag. I fixed tests in the commit [76a3a8e](https://github.com/pesto-students/batch-8-CODERHOOD/commit/76a3a8e5313c558c039ec580beb6ca427daeace3)